### ISSUE TITLE
[Feature Request] Add link to user on main page

### ISSUE THREAD

**Issue by Ascor8522** [InternationalScratchWiki/scratch-confirmaccount-v3#102]
For now, on the main page listing all active requests, you can only go to the requests's detail pages by using the link in the "action" section
However, the page also shows the username of the user who performed the request. Unfortunately, it doesn't act as a link to the user's Scratch profile.
For that, you need to first go the request's detail page, then click the link to go to the user's Scratch profile.

Would be nice if the main page could do that do.
Basically turning text into a link.
